### PR TITLE
fix(portal): Add page_title to all portal pages

### PR DESCRIPTION
**Pull request by bmanifold**
Why:

* When navigating around the portal, the title in the browser tab would
      not show the accurate title of the current page.  This commit adds
      `page_title` to all pages.  The value of the page title has been
      choosen to correspond with the portal's left hand nav menu.

Additional:
* Along with the page titles, the `vertical_table` component was updated to make the left hand headers use a class of `w-1/5` for consistency across pages and to move the info a little further left on each page to try and align it closer with other info on the page.

Here's an example of before and after:

before

after

### CODE REVIEW

**Review by jamilbk — APPROVED** [review]
LGTM. What do you think of adding a `• Firezone` suffix to each page title like we do for docs / website? E.g. `Sites • Firezone`

**Comment by bmanifold** [comment]
@jamilbk, it's already there: https://github.com/firezone/firezone/blob/ce932ffd90f0746aebba8d1b2e6a819c891794b7/elixir/apps/web/lib/web/components/layouts/root.html.heex#L17
😄

**Comment by jamilbk** [comment]
> it's already there:

Always one step ahead!

File: elixir/apps/web/lib/web/live/actors/edit.ex [hunk 1]
```diff
@@ -17,7 +17,7 @@ defmodule Web.Actors.Edit do
           actor: actor,
           groups: groups,
           form: to_form(changeset),
-          page_title: "Edit actor #{actor.name}"
```
Comment (AndrewDryga) [inline elixir/apps/web/lib/web/live/actors/edit.ex:20]: I think it's better to have an explicit title for edit/new/etc pages, imagine you try to edit multiple users at once - you open a few tabs, and they all will be named "Actors" while if we keep the old title you can tell which actor is there by the browser tab name.

File: elixir/apps/web/lib/web/live/policies/edit.ex [hunk 2]
```diff
@@ -10,7 +10,15 @@ defmodule Web.Policies.Edit do
...
          nil <- policy.deleted_at do
       form = to_form(Policies.Policy.Changeset.update(policy, %{}))
-      socket = assign(socket, policy: policy, page_title: "Edit Policy", form: form)
+
+      socket =
+        assign(socket,
+          policy: policy,
+          page_title: "Edit Policy",
+          form: form,
+          page_title: "Policies"
```
Comment (AndrewDryga) [inline elixir/apps/web/lib/web/live/policies/edit.ex]: Duplicate assign

Comment (bmanifold) [reply elixir/apps/web/lib/web/live/policies/edit.ex]: Whoops!  Good catch!

File: elixir/apps/web/lib/web/live/actors/edit.ex (hunk 1 above)
Comment (bmanifold) [reply elixir/apps/web/lib/web/live/actors/edit.ex:20]: Happy to discuss this more, but I feel like if that's the case, the likelihood that you'd be able to read the actual browser tab titles is pretty low.  If you're used to opening multiple tabs for Firezone, you're probably opening a lot of other tabs as well, which means the tab titles get to be pretty small and you wouldn't be able to see the actor name.

All that said, I'm not hard against this, just had a gut feeling it wouldn't add too much value.
@jamilbk, do you have a preference on this?

Comment (jamilbk) [reply elixir/apps/web/lib/web/live/actors/edit.ex:20]: Hmm no super strong preference, but I do lean towards keeping them action-scoped at least. That way if I'm working to change a lot of actors/sites/etc (and I don't or can't use the REST API) I can open them in tabs and at least have some chance of seeing which tabs are which.

How about this?

- show view: `"#{record.name || "Site #{record.id}"} • Firezone"`
- edit view: `"Edit #{record.name || "Edit site"} • Firezone"`
- index view: `"Sites • Firezone"`
- new view: `"New Site • Firezone"`

Comment (bmanifold) [reply elixir/apps/web/lib/web/live/actors/edit.ex:20]: Ok, sounds good.  I must be in the minority of not really looking at my browser tab titles 😄 
I'll get the PR updated

**Pull request merged**